docs/usage/custom_apps.md

```diff
@@ -53,6 +53,15 @@ def configure_app
       access_token: "the_token_for_your_custom_app_found_in_admin"
     )
 
+  ShopifyAPI::Context.setup(
+    api_key: "<api-key>",
+    api_secret_key: "<api-secret-key>",
+    scope: "read_orders,read_products,etc",
+    is_embedded: true, # Set to true if you are building an embedded app
+    api_version: "2024-01", # The version of the API you would like to use
+    is_private: false, # Set to true if you have an existing private app
```

Comment:
What does `is_private` do? 
Should it be set to true for these app types?

Reply:
yup, i think you're right